circus-train-aws-sns/src/test/java/com/hotels/bdp/circustrain/aws/sns/event/SnsListenerTest.java

```diff
@@ -53,15 +56,19 @@
   private static final String SUBJECT = "choochoo";
   private static final RuntimeException ERROR = new RuntimeException("error message");
   private static final String ENDTIME = "endtime";
-  private static List<EventPartition> PARTITIONS_0;
-  private static List<EventPartition> PARTITIONS_1;
+  private static EventPartition PARTITION_0;
+  private static EventPartition PARTITION_1;
+
   private static final String STARTTIME = "starttime";
   private static final String EVENT_ID = "EVENT_ID";
+  private static final String PROTOCOL_VERSION = "\"protocolVersion\":\"2.0.0\"";
```

Comment:
can you change this to be just the number and put `"protocolVersion"` back in the string? It harder to read like this

Reply:
Sure, my thinking was this was less repeated text to type but I agree it's not as easy to read and the part that is going to change is the number, not the rest of it. Will change like you suggest.